lib/acquia-http-hmac/rack_authenticate.rb

```diff
@@ -23,6 +23,8 @@ def call(env)
         return unauthorized if auth_header.empty?
 
         attributes = Acquia::HTTPHmac::Auth.parse_auth_header(auth_header)
+        # Put the id into a variable regardless of authentication so we can read/log it later
+        env['REQUESTED_HMAC_ID'] = attributes[:id]
```

Comment:
@pdrakeweb Didn't want to cause confusion because only the final 2 values are prefixed with `ACQUIA-HTTP-HMAC` and imo `ACQUIA-HTTP-HMAC-REQUESTED-HMAC-ID` or something like that would have been ambiguous.